### DISCUSSION TITLE
What is the cost for a custom wake word?

### DISCUSSION THREAD

**Discussion by X-Ryl669** [toverainc/willow#430] [General]
I don't know if you've contacted Espressif for custom wake word customization, but if you did, I'd like to know how much would it cost to implement one?

**Comment by kristiankielhofner** [comment]
In our case, free.

Our focus with Willow is and continues to be quality, stability, speed, and functionality - things that contribute to the actual user experience in the real world. We completely understand why many users are interested in custom wake words but in the end they don't impact any of these core goals. Our focus is how well Willow works - not what you call it.

That said we have plans to support more wake words and potentially custom wake words at some point, it's just near the bottom of the list in terms of priority because we still have a lot of work to do to realize our goals of matching commercial solutions and eventually surpassing them.
